.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
This has nothing to do with the pull request as a whole. Split such things into different pull requests so that discussion and review can stay focused.

In this case, I am not really interested in including this file, as none of the editors I use recognize it, and I don't want to accumulate tooling cruft that I have no relation to in my projects.